admin/class-gdpr-telemetry.php

```diff
@@ -391,18 +392,21 @@ private static function _face_detect( $path ) {
 		}
 
 		/* Search for plugin */
-		if ( $data = self::_localize_plugin( $path ) ) {
+		$data = self::_localize_plugin( $path );
+		if ( $data ) {
 			return array(
 				'type' => 'Plugin',
 				'name' => $data['Name'],
 			);
-
-		/* Search for theme */
-		} else if ( $data = self::_localize_theme( $path ) ) {
-			return array(
-				'type' => 'Theme',
-				'name' => $data->get( 'Name' ),
-			);
+		} else {
```

Comment:
This seems more readable with an `else if`

Reply:
You can't assign a variable in a conditional scope.

**Bad**
```PHP
if ( $data = self::_localize_theme( $path ) ) {
```

**Good**
```PHP
$data = $data = self::_localize_theme( $path );

if ( $data ) {
```